Test/DependencyCollector/FunctionalTests/FuncTest/Helpers/DeploymentAndValidationTools.cs

```diff
@@ -197,11 +197,17 @@ public static void CleanUp()
         public static void Validate(
             TelemetryItem<RemoteDependencyData> itemToValidate,
             TimeSpan accessTimeMax,
-            bool successFlagExpected)
+            bool successFlagExpected,
+            string resultCodeExpected = "DontCheck")
```

Comment:
Can you clarify this? Do we have tests that do not check the result code? If so - maybe it's a good idea to modify every test?